### PR TITLE
feat: add option to display author for writeups

### PR DESCRIPTION
**Pull request by Stefan-Radu**
very easy to implement. the link on author leads to the link put in .config. we could also do it such that the link leads to about, but I think it wouldn't make much sense.

this is how it would look like:
image

what do you think?

ps: this is rebased into #8 (so I could get the authors)

### CODE REVIEW

**Review by Mathis-Dory — APPROVED** [review]
Very good idea I like it thx :)

**Comment by Mathis-Dory** [comment]
Maybe we should wait for the merge of #22 before merging this one.

**Comment by Stefan-Radu** [comment]
> Maybe we should wait for the merge of #22 before merging this one.

I proposed to merge this one into #22

**Comment by Mathis-Dory** [comment]
> > Maybe we should wait for the merge of #22 before merging this one.
> 
> 
> 
> I proposed to merge this one into #22 

Perfect merged it 😀

**Pull request merged**
